perma_web/perma/tasks.py

```diff
@@ -1902,12 +1925,21 @@ def confirm_added_metadata_to_existing_daily_item_files(file_ids):
 
         expected_metadata = InternetArchiveFile.standard_metadata_for_link(link)
         try:
-            assert expected_metadata.items() <= ia_file.metadata.items()
+            assert expected_metadata.items() <= ia_file.metadata.items(), f"{expected_metadata.items()} != {ia_file.metadata.items()}"
         except AssertionError:
             # modify_xml tasks can take some time to complete;
             # the task for this link appears not to have finished yet.
             # we need to check again later.
-            file_ids_to_check_again.append(file_id)
+            print(previous_attempts)
```

Comment:
Do you want to leave the print statement here?

Reply:
Nope. Thanks!